boxsdk/object/watermark.py

```diff
@@ -0,0 +1,9 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+
+from .base_endpoint import BaseEndpoint
+
+
+class Watermark(BaseEndpoint):
```

Comment:
This class doesn't look like it gets used anywhere.  Should we potentially make this just an API JSON object and use it to hold the watermark info instead of returning a `dict` in the item methods?